src/runtime/GlobalObjectBuiltinArray.cpp

```diff
@@ -1751,6 +1751,16 @@ void GlobalObject::installArray(ExecutionState& state)
     m_arrayPrototype->setPrototype(state, m_objectPrototype);
     m_arrayPrototype->defineOwnProperty(state, ObjectPropertyName(state.context()->staticStrings().constructor), ObjectPropertyDescriptor(m_array, (ObjectPropertyDescriptor::PresentAttribute)(ObjectPropertyDescriptor::WritablePresent | ObjectPropertyDescriptor::ConfigurablePresent)));
 
+    Object* blackList = new Object(state);
+    blackList->setPrototype(state, Value(Value::Null));
+    blackList->defineOwnProperty(state, ObjectPropertyName(state.context()->staticStrings().copyWithin), ObjectPropertyDescriptor(Value(true)));
+    blackList->defineOwnProperty(state, ObjectPropertyName(state.context()->staticStrings().entries), ObjectPropertyDescriptor(Value(true)));
+    blackList->defineOwnProperty(state, ObjectPropertyName(state.context()->staticStrings().fill), ObjectPropertyDescriptor(Value(true)));
+    blackList->defineOwnProperty(state, ObjectPropertyName(state.context()->staticStrings().find), ObjectPropertyDescriptor(Value(true)));
+    blackList->defineOwnProperty(state, ObjectPropertyName(state.context()->staticStrings().findIndex), ObjectPropertyDescriptor(Value(true)));
+    blackList->defineOwnProperty(state, ObjectPropertyName(state.context()->staticStrings().keys), ObjectPropertyDescriptor(Value(true)));
+    blackList->defineOwnProperty(state, ObjectPropertyName(state.context()->staticStrings().values), ObjectPropertyDescriptor(Value(true)));
```

Comment:
Each property of blacklist should have ` PropertyDescriptor{[[Value]]: V, [[Writable]]: true, [[Enumerable]]: true, [[Configurable]]: true}`, but default parameter of [ObjectPropertyDescriptor](https://github.com/pando-project/escargot/blob/7f7f317340793ed2b593c8537c9c98ecaa1f58e8/src/runtime/Object.h#L233) would set the `NotPresent` attibute which is wrong.

Reply:
Please fix this code as follow.
`ObjectPropertyDescriptor(Value(true), ObjectPropertyDescriptor::AllPresent)`